plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1768,15 +1741,6 @@ fc::time_point producer_plugin_impl::calculate_pending_block_time() const {
    return block_time;
```

Comment:
Should return `block_timestamp_type`. One way to implement:

```cpp
return block_timestamp_type((base.time_since_epoch().count() / config::block_interval_us) + 1);
```

Reply:
addressed by [dce7f41](https://github.com/AntelopeIO/leap/pull/867/commits/dce7f418c3c98b51e7ad6e5235dd84f35cbd034c)